openedx_tagging/core/tagging/models/system_defined.py

```diff
@@ -1,4 +1,6 @@
 """ Tagging app system-defined taxonomies data models """
+from __future__ import annotations
+
 import logging
 from typing import Any, List, Type, Union
```

Comment:
```suggestion
from typing import Any, Type, Union
```
I think we have an unused import here.